modules/reference/pages/cluster-properties.adoc

```diff
@@ -2467,11 +2456,15 @@ If `null`, the property is disabled, and traffic is not limited.
 
 *Default:* `null`
 
+*Related topics*:
+
+* xref:manage:cluster-maintenance/manage-throughput.adoc#node-wide-throughput-limits[Node-wide throughput limits]
+
 ---
 
 === kafka_throughput_replenish_threshold
 
-Threshold for refilling the token bucket. Will be clamped between 1 and kafka_throughput_limit_node_*_bps.
+Threshold for refilling the token bucket. Will be clamped between 1 and `kafka_throughput_limit_node_*_bps`.
```

Comment:
Not sure if "clamped" is standard terminology for devs or at least in Kafka

Reply:
Not that I know of. Bad grammar would be my guess